app/src/main/kotlin/com/simplemobiletools/calendar/pro/activities/SettingsActivity.kt

```diff
@@ -621,6 +623,34 @@ class SettingsActivity : SimpleActivity() {
         }
     }
 
+    private fun setupAddBirthdaysAutomatically() {
```

Comment:
oh, this whole thing shouldnt be a settings item, just a checkbox in the importing dialog

Reply:
Ok, so just to confirm:
- user taps on "Add contact birthdays" from menu
- dialog with setting reminders is displayed, and despite reminders user also has a checkbox "Add new birthdays automatically"
- if checkbox is selected, we also store the reminders setting in config
- rest works the same as now - user starts an app, and we check for new birthdays
Of course, the same logic would be applied to anniversaries too.
Am I right?

Reply:
yep, like that